packages/appChrome/stories/AppChrome.stories.tsx

```diff
@@ -84,7 +91,35 @@ export const SidebarWithItemsWithOnClick = args => (
 );
 
 export const SidebarWithItemsActive = args => (
-  <Sidebar isOpen={true}>
+  <Sidebar isOpen={true} {...args}>
+    <SidebarSection label="Section header">
+      <SidebarItem url="http://google.com" isActive={true} key="one">
+        <SidebarItemLabel>Lorem Ipsum</SidebarItemLabel>
+      </SidebarItem>
+      <SidebarItem url="http://google.com" key="two">
+        <SidebarItemLabel>Dolor Sit</SidebarItemLabel>
+      </SidebarItem>
+      <SidebarItem url="http://google.com" key="three">
+        <SidebarItemLabel>Amet Consecutor</SidebarItemLabel>
+      </SidebarItem>
+      <SidebarItem url="http://google.com" key="four">
+        <SidebarItemLabel>Adipiscing Edit</SidebarItemLabel>
+      </SidebarItem>
+      <SidebarItem url="http://google.com" key="five">
+        <SidebarItemLabel>Praesent Massa</SidebarItemLabel>
+      </SidebarItem>
+    </SidebarSection>
+  </Sidebar>
+);
+
+export const SidebarWithCustomSidebarColors = args => (
+  <Sidebar
+    isOpen={true}
+    bgColor={"#000"}
+    hoverColor={"#988ee4"}
+    activeColor={"#8375eb"}
+    {...args}
```

Comment:
Should we try to set up this story so that the user can change these colors with the Storybook controls?